docker/tool/linux/run.ps1

```diff
@@ -1,5 +1,5 @@
 # Get IoTEdgeDev source folder
-$source_folder = Get-Location | Split-Path | Split-Path 
+$source_folder = Get-Location | Split-Path | Split-Path | Split-Path
 
 # Run Docker Container
 docker run -it -v //var//run//docker.sock://var//run//docker.sock -v ${source_folder}:/home/iotedge/tool microsoft/iotedgedev:latest-linux
```

Comment:
When I run the container I map to /home/iotedge root because that is where I want to store files by default.

1. keep the tool directory mapping
2. add a second mapping that is the working directory...the default directory that the user is dropped into when they run the container.

Reply:
I realized there are two personas that was diff mapped drives.

1. Tool developer who needs to run tox in container
2. End user who wants to map /home/iotedge to a local folder.

Can you create a diff run file for each?  

One that maps current folder to /home/iotedge for #1 above
One that maps current folder to /home/iotedge/tool for #2 above.